### PR TITLE
docs: update document renderer

### PR DESCRIPTION
**Pull request by Nebulis**
Thanks to the feedback provided in https://spectrum.chat/openattestation/general/oa-docs-custom-renderer-creating-document-renderer-section-need-to-improve~4eb49ed0-b55b-4964-a859-c5d5a5df1be0, the following changes has been made:
- the flow of the creation is
  - define data
  - create a template
  - configure the template
  - configure the renderer
- to avoid any confusion (the storybook picture showed 2 templates) we start by removing the provided template example and build one
- all references `customXX` have been replaced by `cocXX`
- added story creation
- explanation of the properties in the template configuration

### CODE REVIEW

File: docs/advanced/custom-renderer.md [hunk 1]
```diff
@@ -193,11 +152,79 @@ export const CustomTemplate: FunctionComponent<
...
+```
+
+We can now [start storybook](#run-development-preview) and make sure our component looks like expected.
 
 ![Completed Story Book View](/docs/advanced/custom-renderer/completed-storybook.png)
 
-Once that template code has been updated, you will see that the storybook component has been updated to reflect the change.
+### Certificate of Completion template configuration
+
+An OA document may have multiple views, each of them rendered in separate tabs. For example, an OA document that is a degree certificate may have the actual certificate as one view, and the transcript as another view in a single template. A demo of the multiple views feature can be found [here](https://opencerts.io/?q={%22type%22:%22DOCUMENT%22,%22payload%22:{%22uri%22:%22https://opencerts.io/static/demo/mainnet.opencerts%22,%22permittedActions%22:[%22STORE%22],%22redirect%22:%22https://opencerts.io%22}}).
```
Comment (gjj) [inline docs/advanced/custom-renderer.md:188]: I believe this link was changed from `https://opencerts.io/static/demo/mainnet.opencerts` to `https://opencerts.io/static/demo/homestead.opencert`

Comment (Nebulis) [reply docs/advanced/custom-renderer.md:188]: oh yeah good catch

File: docs/advanced/custom-renderer.md [hunk 2]
```diff
@@ -55,23 +55,23 @@ This is a live preview where you can see the changes when you:
...
 ### Update sample document data and type
 
 To update the raw document data and the corresponding data type, you will need to update the data definition file in `src/templates/sample.ts`:
 
-```js
+```typescript jsx
 import { Document } from "@govtechsg/decentralized-renderer-react-components";
 
-export interface CustomTemplateCertificate extends Document {
+export interface CocTemplateCertificate extends Document {
```
Comment (gjj) [inline docs/advanced/custom-renderer.md:67]: I followed through the step and ran into an error, which was traced back to this.

Reason being: Integration tests make use of this, specifically this line `import { customTemplateCertificate } from "../templates/sample";` is causing the build error.

We could add an instruction to delete the integration tests as well.

Comment (gjj) [reply docs/advanced/custom-renderer.md:67]: Specifically, this is the file: https://github.com/gjj/test-new-demo/blob/ebbcb3ae63183436717435cecf6aa81553f6aaa7/src/integration/integration.spec.ts

The error on Netlify (and locally is the same too): https://app.netlify.com/sites/nifty-noether-8b2f72/deploys/5f27f96d0b21193aea00ad6a

Comment (Nebulis) [reply docs/advanced/custom-renderer.md:67]: error at which step in the doc ? build ?

Comment (gjj) [reply docs/advanced/custom-renderer.md:67]: yep, either (a) `npm run build` locally or (b) on Netlify, and that's cos of the integration tests.

File: docs/advanced/custom-renderer.md (hunk 1 above)
Comment (Nebulis) [reply docs/advanced/custom-renderer.md]: fixed :)

File: docs/advanced/custom-renderer.md (hunk 2 above)
Comment (Nebulis) [reply docs/advanced/custom-renderer.md:75]: Added a step to clean the repo first

**Pull request merged**